src/cdk-experimental/listbox/listbox.ts

```diff
@@ -89,19 +93,58 @@ export class CdkOption implements ListKeyManagerOption, Highlightable {
     this._active = false;
```

Comment:
Just noticed this incidentally while looking at something else: `activate()` checks `_isInteractionDisabled`, but `deactivate` doesn't